src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -107,22 +111,19 @@ internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSynt
                                 cancellationToken.ThrowIfCancellationRequested();
 
                                 var fixData = await reference.GetFixDataAsync(document, node, placeSystemNamespaceFirst, cancellationToken).ConfigureAwait(false);
-                                if (fixData != null)
-                                {
-                                    var codeAction = CreateCodeAction(document, fixData);
-                                    context.RegisterCodeFix(codeAction, diagnostic);
-                                    count++;
-                                }
+                                var codeAction = TryCreateCodeAction(document, fixData);
```

Comment:
Can `fixData` not be null anymore?  `TryCreateCodeAction` doesn't handle that case.